UnitsNet.Tests/GeneratedCode/AccelerationTestsBase.g.cs

```diff
@@ -52,76 +52,76 @@ namespace UnitsNet.Tests
     [TestFixture]
 // ReSharper disable once PartialTypeWithSinglePart
     public abstract partial class AccelerationTestsBase
-    {
-        protected abstract double CentimeterPerSecondSquaredInOneMeterPerSecondSquared { get; }
-        protected abstract double DecimeterPerSecondSquaredInOneMeterPerSecondSquared { get; }
-        protected abstract double KilometerPerSecondSquaredInOneMeterPerSecondSquared { get; }
-        protected abstract double MeterPerSecondSquaredInOneMeterPerSecondSquared { get; }
-        protected abstract double MicrometerPerSecondSquaredInOneMeterPerSecondSquared { get; }
-        protected abstract double MillimeterPerSecondSquaredInOneMeterPerSecondSquared { get; }
-        protected abstract double NanometerPerSecondSquaredInOneMeterPerSecondSquared { get; }
-
-// ReSharper disable VirtualMemberNeverOverriden.Global
-        protected virtual double CentimeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double DecimeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double KilometerPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double MeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double MicrometerPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double MillimeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double NanometerPerSecondSquaredTolerance { get { return 1e-5; } }
+    {

```

Comment:
I think maybe you have configured your git client to enforce a certain end of line, so when you called `generate-code.bat` it replaced all lines of all files with almost identical ones.